src/components/menu/menu-profile.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+import * as React from 'react';
+// Components
+import {
+  ListProfile as StyledListProfile,
+  ListItemProfile as StyledListItemProfile,
+  ProfileImgContainer as StyledProfileImgContainer,
+  ProfileImg as StyledProfileImg,
+  ProfileLabelsContainer as StyledProfileLabelsContainer,
+  ProfileTitle as StyledProfileTitle,
+  ProfileSubtitle as StyledProfileSubtitle,
+  ProfileBody as StyledProfileBody,
+} from './styled-components';
+import {mapStyletronProps} from './utils';
+import {getOverride} from '../../helpers/overrides';
+// Types
+import type {StatelessMenuProfilePropsT} from './types';
+
+export default function MenuProfile({
+  items,
+  getProfileItemLabels,
+  getProfileItemImg,
+  getProfileItemImgText,
+  getRequiredItemProps = (item, index) => ({key: String(index)}),
+  rootRef = React.createRef(),
+  overrides = {},
+}: StatelessMenuProfilePropsT) {
+  const ListProfile = getOverride(overrides.ListProfile) || StyledListProfile;
+  const ListItemProfile =
+    getOverride(overrides.ListItemProfile) || StyledListItemProfile;
+
+  const ProfileImgContainer =
+    getOverride(overrides.ProfileImgContainer) || StyledProfileImgContainer;
+  const ProfileImg = getOverride(overrides.ProfileImg) || StyledProfileImg;
+
+  const ProfileLabelsContainer =
+    getOverride(overrides.ProfileLabelsContainer) ||
+    StyledProfileLabelsContainer;
+  const ProfileTitle =
+    getOverride(overrides.ProfileTitle) || StyledProfileTitle;
+  const ProfileSubtitle =
+    getOverride(overrides.ProfileSubtitle) || StyledProfileSubtitle;
+  const ProfileBody = getOverride(overrides.ProfileBody) || StyledProfileBody;
+
+  return (
+    <ListProfile $ref={rootRef}>
+      {items.map((item, index) => {
+        const requiredProps = getRequiredItemProps(item, index);
+        // $FlowFixMe
+        const {key, ...itemProps} = mapStyletronProps(requiredProps);
```

Comment:
I think $FlowFixMe gets stripped in our dist, so consumers might end up getting warnings for our internal flow errors (at least that's what was happening in #110). Any chance we can avoid the FlowFixMe here?

Reply:
Oh interesting, will update this code. Thanks!